### DISCUSSION TITLE
Virtualized select

### DISCUSSION THREAD

**Discussion by nicooprat** [unovue/reka-ui#948] [Ideas]
Hi there,

Is there any plan to be able to have a virtualized select? https://www.radix-vue.com/components/select.html

We are currently migrating from ElementPlus and this is one of the component that is desperately needed for performances reasons. Docs: https://element-plus.org/en-US/component/select-v2.html#virtualized-select

Looks like Radix only provides virtualization for Listbox for now: https://www.radix-vue.com/components/listbox.html#virtual-list

Thanks!

**Comment by chasegiunta** [comment]
I was wondering the same thing for Select & Combobox

**Comment by zernonia** [comment]
Yup. Both component essentially will be built upon `Listbox`, which would support Virtualization in v2

**Reply by nicooprat** [reply]
Glad to hear! Will this be added to the v2 roadmap? https://github.com/orgs/radix-vue/projects/3?query=is%3Aopen+sort%3Aupdated-desc
